nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -145,6 +151,16 @@
             .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor PASSWORD = new PropertyDescriptor.Builder()
+            .name("Password")
+            .displayName("Password")
+            .description("Password used for decrypting archive entries. Supports Zip files encrypted with ZipCrypto or AES")
+            .required(false)
+            .sensitive(true)
+            .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
```

Comment:
I don't think we want a sensitive property to accept variables (or any EL) as the password `abc${def}` would be ambiguous as to whether it should be interpreted literally or interpolate the value of the variable `def`. Our approach thus far has been not to allow EL in any sensitive property (the introduction of sensitive parameters in 1.10.0 helped with this).

Reply:
That makes sense, I was wondering about that as a potential problem, I will remove support for expression language on this property.